README.md

```diff
@@ -24,25 +23,37 @@ To use the Draft Web Design Standards on your project, you’ll need to include
 
 First, download the Draft Web Design Standards assets:
 
-[https://github.com/18F/web-design-standards/archive/v0.8.3.zip](https://github.com/18F/web-design-standards/archive/v0.8.3.zip).
+[https://github.com/18F/web-design-standards/releases/download/v0.9.0/uswds-0.9.0.zip](https://github.com/18F/web-design-standards/releases/download/v0.9.0/uswds-0.9.0.zip)
 
 Then, add the `dist` directory files into a relevant place in your code base — likely a directory where you keep third-party libraries:
 
 ```
-- uswds
-  - fonts/
-  - img/
-  - js/
-  - css/
+- uswds-0.9.0/
+  - dist/
+    - css/
+    - fonts/
+    - img/
+    - js/
 ```
 
-Refer to these files by adding a `<link>` and a `<script>` element into your HTML pages:
+Refer to these files by adding the following `<link>` and `<script>` elements into your HTML pages.
+
+Add this to your `<head>` element:
 
+```html
+<link rel="stylesheet" href="/path/to/your/assets/css/uswds-0.9.0.css">
 ```
-<link rel="stylesheet" href="/path/to/your/assets/css/main.css">
-<script src="/path/to/your/assets/js/components.js"></script>
+
+Add this before the closing `</body>` tag:
+
+```html
+<script src="/path/to/your/assets/js/uswds-0.9.0.js"></script>
 ```
 
+This version of the Standards includes jQuery version `2.2.0` bundled within the
+JavaScript file. Please make sure that you are not including any other version
```

Comment:
I'd replace "Please make sure that you are not" with "Please make sure you're not" (just a stylistic update to make the tone a wee bit more conversational).